### DISCUSSION TITLE
Image Occlusion Cards

### DISCUSSION THREAD

**Discussion by tedblizzard100** [ObsidianToAnki/Obsidian_to_Anki#174] [General]
It would be amazing to have this capability in the **Obsidian to Anki Plugin.** 

https://twitter.com/rem_note/status/1281683360075538432?lang=en

**Comment by problemadder** [comment]
you could use the anki image occlusion add on. it works great

**Reply by TrillStones** [reply]
> you could use the anki image occlusion add on. it works great

The purpose of using Obsidian to Anki is to fully manage cards in Obsidian rather than maintaining them in two places.
